book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
Also here, we want to make sure we can make a rental, not only read what was rented

- [ ] Kindly update attribute for `rentals`

Reply:
Sure thing. Thank you for your detailed review @AshakaE

Reply:
- Well resolved 👍🏽